package.json

```diff
@@ -7,6 +7,8 @@
     "@mui/icons-material": "^5.15.17",
     "@mui/material": "^5.15.17",
     "@mui/styled-engine-sc": "^6.0.0-alpha.18",
+    "@mui/x-date-pickers": "^7.5.0",
+    "@mui/x-date-pickers-pro": "^7.5.0",
```

Comment:
This one is a paid one, if I'm not mistaken. So, I assume you are not using it anywhere. Feel free to remove it